src/sentry/integrations/slack/notifications.py

```diff
@@ -109,17 +110,26 @@ def _notify_recipient(
         if SiloMode.get_current_mode() == SiloMode.CONTROL:
             post_message_task = post_message_control
 
+        has_sdk_flag = features.has(
+            "organizations:slack-sdk-notify-recipient", notification.organization
+        )
+
```

Comment:
i have to do the backend equivalent of prop drill with this :(

Reply:
its gonna be so much nicer once we can remove the ff and delete all of the flags